v1alpha1/meshmodel.json

```diff
@@ -2027,6 +2147,114 @@
+              "fullType": "string",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "destination",
+              "description": "Destination is the service to allow ingress traffic",
+              "label": "",
+              "type": "IdentityBindingSubject",
+              "longType": "IdentityBindingSubject",
+              "fullType": "easemesh.v1alpha1.IdentityBindingSubject",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "sources",
+              "description": "Sources are the services to allow egress traffic",
+              "label": "repeated",
+              "type": "IdentityBindingSubject",
+              "longType": "IdentityBindingSubject",
+              "fullType": "easemesh.v1alpha1.IdentityBindingSubject",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "rules",
+              "description": "Rules are the traffic rules to allow (HTTPRoutes)",
+              "label": "repeated",
+              "type": "TrafficTargetRule",
+              "longType": "TrafficTargetRule",
+              "fullType": "easemesh.v1alpha1.TrafficTargetRule",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            }
+          ]
+        },
+        {
+          "name": "TrafficTargetRule",
+          "longName": "TrafficTargetRule",
+          "fullName": "easemesh.v1alpha1.TrafficTargetRule",
+          "description": "TrafficTargetRule is the TrafficSpec to allow for a TrafficTarget",
+          "hasExtensions": false,
+          "hasFields": true,
+          "hasOneofs": false,
+          "extensions": [],
+          "fields": [
+            {
+              "name": "kind",
+              "description": "Kind is the kind of TrafficSpec to allow",
+              "label": "",
+              "type": "string",
+              "longType": "string",
+              "fullType": "string",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "name",
+              "description": "Name of the TrafficSpec to use",
+              "label": "",
+              "type": "string",
+              "longType": "string",
+              "fullType": "string",
+              "ismap": false,
+              "isoneof": false,
+              "oneofdecl": "",
+              "defaultValue": ""
+            },
+            {
+              "name": "matches",
+              "description": "Matches is a list of TrafficSpec routes to allow traffic for",
```

Comment:
Is the `matches` a matched name collection of `HTTPRouteGroup`? Could you provide a model relationship diagram to help us understand your design?

Reply:
I've updated the comments to explain `TrafficSpec` and `matches`. Please just review `meshmodel.proto`, other files are all auto-generated.